tests/error/struct_errors/invalid_attribute_assign1.err

```diff
@@ -1,7 +1,8 @@
-Guppy compilation failed. Error in file $FILE:15
+Error: Attribute not found (at $FILE:15:6)
+   | 
+13 | @guppy(module)
+14 | def foo(s: MyStruct) -> None:
+15 |     s.z = 2
+   |       ^ Attribute `z` not found on type `int`
```

Comment:
Would it be an easier read if this error was:
```suggestion
   |       ^ Attribute `z` of type `int` not found
```
or even better
```suggestion
   |       ^ `MyStruct` has no attribute `z` of type `int`
```

I actually don't see the relevance of the inferred type here, the error message on the left is quite descriptive as is. Perhaps I am missing some context here.

Reply:
Oops yes it's supposed to say "MyStruct" instead of "int" 👍